Cargo.toml

```diff
@@ -10,6 +10,9 @@ readme = "README.md"
 [features]
 vendored = ["openssl/vendored"]
 
+# Switch for MacOS to keep support for older versions, Windows targets and openssl target will have alpn regardless
```

Comment:
This feature needs to apply to all backends - OpenSSL 1.0.1 and sufficiently old Windows versions won't support it either.

Reply:
Added alpn feature switch to openssl targets and windows targets.